core/src/test/java/com/sequenceiq/cloudbreak/converter/v2/StackToTemplatePreparationObjectConverterTest.java

```diff
@@ -526,7 +534,17 @@ public void testMissingClouderaManagerIp() {
         when(stackMock.getPrimaryGatewayInstance()).thenReturn(dummyMetadata);
 
         TemplatePreparationObject result = underTest.convert(stackMock);
-        assertEquals("10.0.0.1", result.getGeneralClusterConfigs().getClusterManagerIp());
+
+        assertThat(result.getGeneralClusterConfigs().getClusterManagerIp()).isEqualTo("10.0.0.1");
         verify(gatewayConfigService, times(1)).getPrimaryGatewayIp(any(Stack.class));
     }
+
+    @Test
+    void testRdsSslCertificateFile() {
```

Comment:
**question:** This looks like the only _new_ test in this file, is that the case? (I'm not missing any?)

The rest of the changes are conversions to AssertJ?

Reply:
Correct, the rest is just noise due to AssertJ.